src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializer.cs

```diff
@@ -102,16 +102,12 @@ public static int WriteUnicodeNoEscape(byte[] buffer, int cursor, ushort ordinal
             buffer[cursor++] = unchecked((byte)(0b_1100_0000 | (ordinal >> 6)));
             buffer[cursor++] = unchecked((byte)(0b_1000_0000 | (ordinal & 0b_0011_1111)));
         }
-        else if (ordinal <= 0xFFFF)
+        else // all other <= 0xFFFF which is ushort.MaxValue
```

Comment:
@SimonCropp Seems to be a style issue with how you dropped the comment. Sorry the rules in this repo are ~annoying~ strict 😄